packages/dialtone-css/lib/build/less/recipes/attachment_carousel.less

```diff
@@ -3,7 +3,7 @@
   width: var(--dt-space-1000);
   max-height: 100px; /* stylelint-disable-line meowtec/no-px */
 
-  .d-recipe-attachment-carousel__media-list {
+  &__media-list {
```

Comment:
I was keeping them like `.d-recipe-attachment-carousel__media-list` as it's a lot simplier to find those classes when you're debugging or trying to fix something quickly.

Might seem too verbose, but it's a good practice I read somewhere and made sense for me.

Reply:
Just need to move them to the right place of course

Reply:
I feel like this is one of those debated topics in CSS. writing the whole class name gives it better searchability, but using `&` makes it easier to see what is related.

I made the restructure, let me know if that's what you were thinking.